src/components/Picklist/index.js

```diff
@@ -68,12 +54,16 @@ class Picklist extends Component {
     }
 
     componentDidUpdate(prevProps, prevState) {
-        const { isOpen: wasOpen } = prevState;
+        const { isOpen: wasOpen, dropdownSize } = prevState;
         const { isOpen } = this.state;
         if (!wasOpen && isOpen) {
             this.outsideClick.startListening(this.containerRef.current, () => this.closeMenu());
             this.windowScrolling.startListening(this.handleWindowScroll);
         }
+        const currentSize = this.dropdownRef.current && this.dropdownRef.current.getSize();
+        if (isOpen && JSON.stringify(dropdownSize) !== JSON.stringify(currentSize)) {
+            this.updateDropdownSize(this.dropdownRef.current.getSize());
+        }
```

Comment:
here instead of doing this, we should simply change the bottom default resolvers in InternalOverlay, thus this will work fine always